modules/imaging_uploader/jsx/columnFormatter.js

```diff
@@ -56,6 +56,11 @@ function formatColumn(column, cell, rowData, rowHeaders) {
   }
 
   if (column === 'Tarchive Info') {
+    if (cell === null || cell === "0") {
```

Comment:
`cell` will always be of type `string` or `null`. So, I don't have to check for anything else. I don't know how the imaging uploader works but the redmine ticket 12172 points out zero or null. I just followed it *shrug*